pyvista/plotting/renderer.py

```diff
@@ -3565,7 +3562,7 @@ def add_legend_scale(
             Factor to scale label size relative to title size.
 
         label_format : str, optional
-            A printf style format for labels, e.g. '%E'.
+            A printf style format for labels, e.g. ``'%E'``.
```

Comment:
The Python intersphinx mapping has an item for `old-string-formatting` that maps to https://docs.python.org/3/library/stdtypes.html#old-string-formatting. Do you think it would be helpful to link to this for our users?

Reply:
```suggestion
            A printf style format for labels, e.g. ``'%E'``.
            See `old-string-formatting <https://docs.python.org/3/library/stdtypes.html#old-string-formatting>`_.
```